cdf.toml

```diff
@@ -0,0 +1,39 @@
+[cdf]
+plugins = [
+    "dump",
+    "describe",
+]
+
+featureflags = [
+    "import",
+]
+
+[modules]
+# This is the version of the modules. It should not be changed manually.
+# It will be updated by the 'cdf module upgrade' command.
+version = "0.2.18"
+
+[packages]
```

Comment:
An alternative is to hardcode the packages into the CLI, however, I do not like that option.